cloudbuild.yaml

```diff
@@ -227,14 +227,28 @@ steps:
     - GO111MODULE=on
 
 #
-# Run the e2e tests
+# Run the e2e tests with FeatureGates
 #
 
 - name: 'e2e-runner'
+  args: ['PlayerTracking=true&ContainerPortAllocation=true']
+  id: e2e-feature-gates
   waitFor:
     - push-images
     - build-e2e
 
+#
```

Comment:
This would run one after the other, yes?

Is this where we should be looking at prow, so we can run this in parallel? @roberthbailey ?